BrainPortal/app/helpers/rich_ui_helper.rb

```diff
@@ -261,6 +261,24 @@ def overlay_content_link(name, options = {}, &block)
     html.html_safe
   end
 
+  # Create a link that will open a dialog box.
+  #
+  # Options:
+  #   title: title of the overlay
+  #   link_text: text of the link
+  #   description: content of the overlay
+  #   size: used to set the max-width of the overlay
+  def closable_overlay_content_link(title, link_text, description="", options = {size: "50em"})
```

Comment:
I would re order the arguments: `(link_text, title, description)` makes more sense (look at the HTML code when you use the helper)